.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<h1> Pull Request Template </h1>
+
+<h2> Description </h2>
+
+Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
+* **QA Team** (Quality Assurance) with tests.
+* **reviewers** to understand what are the stakes of the pull request.
+
+Fixes # (issue)
+
+<h2> Type of change </h2>
```

Comment:
Is it also possible to add the target serie(s) to ease the cherry-picking ?

```
<h2> Target serie </h2>

Please delete series that are not relevant.

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x (master)
```

Reply:
Yes :+1:

Reply:
done :)
Thanks